components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java

```diff
@@ -2644,4 +2644,50 @@ public AccessTokenDO retrieveLatestToken(Connection connection, String consumerK
         }
     }
 
+    /**
+     * Revoke access tokens of other tenants when SaaS is disabled.
+     *
+     * @param consumerKey client ID
+     * @param tenantId    application tenant ID
+     * @throws IdentityOAuth2Exception
+     */
+    public void revokeSaaSTokensOfOtherTenants(String consumerKey, int tenantId) throws IdentityOAuth2Exception {
+
+        if (consumerKey == null) {
+            log.error("invalid parameters provided. Client ID: " + consumerKey + "and tenant ID: " + tenantId);
```

Comment:
Start error msg with a capital letter

Reply:
"and tenant ID: " --> " and tenant ID: "
ie. leave a space

Reply:
Already fixed with 0e61801108184e1f3be82f294fe0abe47f09df16